translations/fr.log

```diff
@@ -0,0 +1,97 @@
+*****: fr
+orig->languageName: 'en', 'fr'
+orig->control: 'Control', 'Contrôle'
+orig->print: 'Print', 'Imprimer'
+orig->status: 'Status', 'Statut'
+orig->console: 'Console', 'Console'
+orig->setup: 'Setup', 'Installation'
+orig->current: 'Current°C', 'Actuel°C'
+orig->active: 'Active°C', 'Actif°C'
+orig->standby: 'Standby°C', 'Standby°C'
+orig->move: 'Move', 'Mouvement'
+orig->extrusion: 'Extrusion', 'Extrusion'
+orig->macro: 'Macro', 'Macro'
+orig->stop: 'STOP', 'ARRÊT'
+orig->extruderPercent: 'Extruder%', 'Extrudeuse%'
+orig->speed: 'Speed ', 'Vitesse '
+orig->fan: 'Fan ', 'Ventilo '
+orig->timeRemaining: 'Time left: ', 'Temps Restant: '
+orig->simulated: 'sim'd ', 'Simul. '
+orig->file: 'file ', 'Fichier '
+orig->filament: 'filament ', 'filament '
+orig->slicer: 'slicer ', 'slicer '
+orig->notAvailable: 'n/a', 'n/a'
+orig->pause: 'Pause', 'Pause'
+orig->babystep: 'Baby step', 'Baby step'
+orig->resume: 'Resume', 'Reprise'
+orig->cancel: 'Cancel', 'Annuler'
+orig->reprint: 'Print again', 'Print again'
+orig->resimulate: 'Simulate again', 'Simulate again'
+orig->set: 'Set', 'Set'
+orig->volume: 'Volume ', 'Volume '
+orig->calibrateTouch: 'Calibrate touch', 'Calibrer touch'
+orig->mirrorDisplay: 'Mirror display', 'Affichage en négatif'
+orig->invertDisplay: 'Invert display', 'Inverser affichage'
+orig->theme: 'Theme', 'Théme'
+orig->brightnessDown: 'Brightness -', 'Luminosité -'
+orig->brightnessUp: 'Brightness +', 'Luminosité +'
+orig->saveSettings: 'Save settings', 'Sauver paramêtres'
```

Comment:
"Sauver les paramêtres"
If enough space

Reply:
Unfortunately there isn't enough space.

Done.

Reply:
value is not used anyway. going to delete it in future.